src/Psy/ColorMode.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of Psy Shell.
+ *
+ * (c) 2012-2015 Justin Hileman
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Psy;
+
+/**
+ * A configuration object that represents one of 3 available color modes:
```

Comment:
CS checks failing because this is not a valid short description. Provide one first, then leave a newline, and add this description.